src/platform/plugins/shared/unified_doc_viewer/public/components/doc_viewer_logs_overview/sub_components/badges/event_type.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import React from 'react';
+import { css } from '@emotion/react';
+import type { EuiThemeComputed } from '@elastic/eui';
+import { EuiBadge, EuiText, useEuiTheme } from '@elastic/eui';
+import { euiPaletteRed } from '@elastic/eui';
+
+const dataTestSubj = 'unifiedDocViewLogsOverviewEventType';
+
+const badgeCss = (euiTheme: EuiThemeComputed) => css`
+  max-width: calc(${euiTheme.size.base} * 7.5);
+`;
+
+interface EventTypeProps {
+  eventTypeValue: React.ReactNode;
+}
+export function EventType({ eventTypeValue }: EventTypeProps) {
+  const euiPaletteRed9 = euiPaletteRed(14);
```

Comment:
You can place it outside the component, this could potentially cause extra renders.